### PR TITLE
 Bug 1705753: oc explain does not work for samples operator config CRD

### PR DESCRIPTION
**Pull request by gabemontero**
/hold 

@openshift/sig-developer-experience fyi

given the 1.13 bump we've negotiated dropping this post 4.1.0 GA

but we can start getting some PR testing done and any review folks have bandwidth form in the interim

this also includes the commits from yet to have merged https://github.com/openshift/cluster-samples-operator/pull/143 which just does the k8s 1.13 bump

### CODE REVIEW

**Comment by gabemontero** [comment]
ok clean run now ... yeah, I think possible delays in fsm propagation that were addressed in https://github.com/openshift/cluster-samples-operator/pull/144/commits/446f298b771df6e04e7fc51cca6818fb092dc9c5 could have had some bearing

**Comment by gabemontero** [comment]
/hold cancel

/assign @adambkaplan 

@openshift/sig-developer-experience fyi / ptal as time permits

File: glide.yaml [hunk 1]
```diff
@@ -32,4 +37,19 @@ import:
...
+- package: github.com/pkg/errors
+  version: 645ef00459ed84a119197bfb8d8205042c6df63d
+- package: github.com/spf13/afero
+  version: 588a75ec4f32903aa5e39a2619ba6a4631e28424
+- package: github.com/gobuffalo/envy
+  version: 043cb4b8af871b49563291e32c66bb84378a60ac
+- package: github.com/joho/godotenv
+  version: 5c0e6c6ab1a0a9ef0a8822cba3a05d62f7dad941
+- package: github.com/rogpeppe/go-internal
+  version: 438578804ca6f31be148c27683afc419ce47c06e
```
Comment (adambkaplan) [inline glide.yaml]: Do we need to pin all of these?

**Comment by gabemontero** [comment]
On Wed, May 15, 2019 at 10:54 AM Adam Kaplan <notifications@github.com>
wrote:

> *@adambkaplan* requested changes on this pull request.
>
> @gabemontero <https://github.com/gabemontero> please update your git
> history so it is more legible:
>
>    1. Glide changes should be a bump(*) commit, noting that you added
>    library-go sigs-controller-tools
>    2. You can squash the updates to the Makefile and the generated CRD
>    schema.
>
>
Will do.  Admittedly I was curious how deep the bug naming convention went
a while back and forgot to update this when I got a better feel.  Thanks.

>
>    1.
>
> ------------------------------
>
> In glide.yaml
> <https://github.com/openshift/cluster-samples-operator/pull/144#discussion_r284297800>
> :
>
> > +  version: b19bf474d317b857955b12035d2c5acb57ce8b01
> +# the following copied version from cluster-openshift-controller-manager-operator glide.loc
> +- package: github.com/ghodss/yaml
> +  version: c7ce16629ff4cd059ed96ed06419dd3856fd3577
> +- package: github.com/markbates/inflect
> +  version: d582c680dc4d29c2279628ae00e743005bfcd4fe
> +- package: github.com/pkg/errors
> +  version: 645ef00459ed84a119197bfb8d8205042c6df63d
> +- package: github.com/spf13/afero
> +  version: 588a75ec4f32903aa5e39a2619ba6a4631e28424
> +- package: github.com/gobuffalo/envy
> +  version: 043cb4b8af871b49563291e32c66bb84378a60ac
> +- package: github.com/joho/godotenv
> +  version: 5c0e6c6ab1a0a9ef0a8822cba3a05d62f7dad941
> +- package: github.com/rogpeppe/go-internal
> +  version: 438578804ca6f31be148c27683afc419ce47c06e
>
> Do we need to pin all of these?
>
yes we do....this arose from an iterative meets min to get to compile

File: glide.yaml (hunk 1 above)
Comment (gabemontero) [reply glide.yaml]: yes we do....this arose from an iterative meets min to get to compile

**Comment by gabemontero** [comment]
@adambkaplan reworked commits per request .... PTAL

**Pull request merged**